client/reconfiguration/src/default_handlers.cpp

```diff
@@ -71,20 +82,27 @@ bool ClientTlsKeyExchangeHandler::execute(const State& state, WriteState&) {
   if (master_key.empty()) master_key = psm_.decryptFile(master_key_path_).value_or(std::string());
   if (master_key.empty()) LOG_FATAL(getLogger(), "unable to read the node master key");
```

Comment:
The master_key parameter should not be passed as an argument to exchangeTlsKeys - it is the same in both cases and is not used outside. Generate it in exchangeTlsKeys.

Reply:
done. thanks.